tests/e2e/cypress/integration/help.spec.js

```diff
@@ -0,0 +1,54 @@
+const agencies = require('../fixtures/transit-agencies.json')
+
+describe("Help page spec", () => {
+  beforeEach(() => {
+    cy.visit("/")
+  })
+
+  it("Clicking on Help takes user to Help", () => {
+    cy.contains("Help")
+      .click()
+
+    cy.location("pathname").should("eq", "/help")
+  })
+
+  it("Contains a back link", () => {
+    cy.contains("Help")
+      .click()
+
+    cy.contains("Go back")
+      .then(($e) => {
+        expect($e).attr("href").eql("/")
+      })
+  })
+
+  it("Allows user to go back", () => {
+    cy.contains("Help")
+      .click()
+
+    cy.contains("Go back")
+      .click()
+
+    cy.location("pathname").should("eq", "/")
+  })
+
+  it("Has help information for all transit agencies", () => {
+    cy.contains("Help")
+      .click()
+
+    cy.contains(agencies[0].long_name)
+    cy.contains(agencies[1].long_name)
+    cy.contains(agencies[0].phone)
+    cy.contains(agencies[1].phone)
```

Comment:
Instead of getting `agencies[0]` and `agencies[1]` specifically, would it be possible to loop over all the agencies present in the fixture? This would ensure we are asserting the `long_name` and `phone` are present for any number of agencies.